### ISSUE TITLE
Feature request: Encryption at rest for DAX

### ISSUE THREAD

**Issue by ewbankkit** [terraform-providers/terraform-provider-aws#5500]
Encryption at rest [just released](https://aws.amazon.com/about-aws/whats-new/2018/08/amazon-dynamodb-accelerator--dax--adds-support-for-encryption-at/) for Amazon DynamoDB Accelerator (DAX).
Add support to the [`aws_dax_cluster`](https://www.terraform.io/docs/providers/aws/r/dax_cluster.html) resource.

**Comment by ewbankkit** [comment]
Proposed Terraform syntax is to match the `aws_dynamodb_table` resource's [`server_side_encryption`](https://www.terraform.io/docs/providers/aws/r/dynamodb_table.html#server_side_encryption-1) attribute:

```hcl
resource "aws_dax_cluster" "test" {
  ...

  server_side_encryption {
    enabled = true
  }
}
```

**Comment by bflad** [comment]
Support for this has been merged into master and will release with version 1.33.0 of the AWS provider, likely middle of next week.

**Comment by bflad** [comment]
This has been released in [version 1.33.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1330-august-22-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
